lib/mayaUsd/ufe/UsdPointInstanceModifierBase.h

```diff
@@ -153,23 +185,58 @@ class MAYAUSD_CORE_PUBLIC UsdPointInstanceModifierBase
 
         const size_t instanceIndex = static_cast<size_t>(_instanceIndex);
 
-        PXR_NS::UsdAttribute usdAttr = _getOrCreateAttribute();
-        if (!usdAttr) {
-            return false;
+        // Once setValue() is called, any point instance batch under
+        // construction is closed.  If no batch was created, make a trivial,
+        // unshared batch of a single point instance.
+        _closeBatch();
```

Comment:
The first time you set a value means that batch creation is over for that command, for that point instancer.  All point instance modifiers that share the batch will now use it.